presto-main/src/test/java/com/facebook/presto/sql/query/TestGrouping.java

```diff
@@ -42,5 +42,13 @@ public void testImplicitCoercions()
         assertions.assertQuery(
                 "SELECT GROUPING(k), SUM(v) + 1.0 FROM (VALUES (1, 1)) AS t(k,v) GROUP BY k",
                 "VALUES (0, 2.0)");
+
+        assertions.assertQuery(
+                "SELECT\n" +
+                        "    1.0 * count(*),\n" +
+                        "    grouping(x)\n" +
+                        "FROM (VALUES 1) t(x)\n" +
```

Comment:
Nit (and sorry i didn't notice earlier) you could replace `\n` (and fold it a bit), e.g.

```
assertions.assertQuery(
                 "SELECT 1.0 * count(*), grouping(x) " +
                         "FROM (VALUES 1) t(x) " +
                         "GROUP BY GROUPING SETS ((x), ())",
                 "VALUES (1.0, 1), (1.0, 0)");
```